### DISCUSSION TITLE
MMC TV [SI]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#385] [Show and tell]
### Channel ID

MMCTV.si

### Stream URL

https://vodstr.rtvslo.si/ava_archive05/_definst_/preroll/intermission.mp4/playlist.m3u8

### Notes

Sometimes Geo-blocked

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
That's an Geo-block message you've found.

**Comment by freearhey** [comment]
https://vodstr.rtvslo.si/ava_archive05/_definst_/preroll/intermission.mp4/playlist.m3u8 (static image)
![image translated](https://github.com/iptv-org/iptv/assets/7253922/3353b472-2dfd-4350-a223-0745b6c32a93)
